lib/awful/tooltip.lua

```diff
@@ -401,16 +394,14 @@ end
 --
 -- The valid table values are:
 --
--- * front
--- * middle
--- * back
---
--- The default is:
---
---    {"front", "back", "middle"}
+-- * "front"
+-- * "middle"
+-- * "back"
```

Comment:
I am not sure. The problem here is that it's not really an enum, it's just some entries in a table. So neither `@propertyvalue` or `@tablerowkey` really work. I am not sure if we should overload the meaning of `@propertyvalue`. This might cause problems later on if we start using the schema to generate auto-completion code for those Lua LSP modules.

What I could do is to add a new tag for the 4-5 places where this problem applies?